lib/kuber_hex/gateways/authorize_net.ex

```diff
@@ -1,4 +1,25 @@
 defmodule Gringotts.Gateways.AuthorizeNet do
+
+  @moduledoc """
+  A module for working with the authorize net payment gateway. The module provides 
+  a set of functions to perform transactions via this gateway for a merchant. 
+  To use this module you need to create an account with the authorize net gateway which will provide you with a
+  'name' and a 'transactionKey'. You will need to configure these keys to use the module
+  functions.
+
+  AuthorizeNet api-reference -> https://developer.authorize.net/api/reference/index.html
+
+  The following set of functions have been provided for a transaction
```

Comment:
Some typos. This would look better in a table.

Reply:
Removed the typos. You mean for the name and description
* Authorize -> "description should be"
|Authorize   |  Description|.
Did this way.